### DISCUSSION TITLE
Best way to get release notes for v1/v2?

### DISCUSSION THREAD

**Discussion by shawnoster** [aws/aws-cli#7333] [General]
Hey all,

What's the best way to get release notes for each version? When evaluating new tooling, it's helpful to know what areas to focus on to accelerate approval and testing.

I'm a big fan of SAM as a tool, but the release process could be more helpful and streamlined. Adding a step to the CI/CD to update the Release page with an Issues query of all closed issues that match the release would be a great start. Or add a step to your sprint ceremonies where someone updates the Release page with that sprint's updates.

This is *especially* for minor version updates, assuming you're following semantic versioning (and these days... who doesn't...). For example, why bump from 2.7 to 2.8?

If I'm missing something, I'm happy to [eat crow](https://en.wikipedia.org/wiki/Eating_crow)!

Cheers

**Comment by aBurmeseDev** [comment]
Hi @shawnoster, thanks for reaching out.

I would refer you to check out our [CHANGELOG](https://github.com/aws/aws-cli/blob/v2/CHANGELOG.rst) which includes release notes for each version release. Here are the links for [v1](https://github.com/aws/aws-cli/blob/develop/CHANGELOG.rst) and [v2](https://github.com/aws/aws-cli/blob/v2/CHANGELOG.rst). Please let us know if that's what you're looking for.

Best,
John
